test/test_modules.py

```diff
@@ -528,6 +528,99 @@ def check_backward(cpu_output, gpu_output):
                         check_backward(cpu_output, gpu_output)
 
 
+    @modules(module_db)
+    def test_memory_format(self, device, dtype, module_info):
+        module_cls = module_info.module_cls
+        module_inputs = module_info.module_inputs_func(module_info, device=device, dtype=dtype,
+                                                       requires_grad=False)
+        is_conv = 'Conv' in module_cls.__name__
```

Comment:
Hm no way to push this into the ModuleInfo entry for Conv* somehow?

Reply:
my bad, added a new flag `module_memformat_affects_out` to `ModuleInfo` for this, is that ok?

Reply:
sure I'm good with a new flag! Thanks :)